content/en/real_user_monitoring/_index.md

```diff
@@ -128,13 +128,13 @@ All Datadog SDKs traffic is transmitted over SSL (default 443) to the following
 
 ## Explore Datadog RUM
 
-Access RUM by navigating to [**UX Monitoring > RUM Applications**][1].
+Access RUM by navigating to [**UX Monitoring > Real User Monitoring**][1].
```

Comment:
can we change the hyperlink to go to https://app.datadoghq.com/rum/performance-monitoring ?